src/App.js

```diff
@@ -1,28 +1,27 @@
-import React, { Component } from 'react'
-import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
-import { connect } from 'react-redux';
+import React, {Component} from 'react';
+import {BrowserRouter as Router, Route, Switch} from 'react-router-dom';
+import {connect} from 'react-redux';
 import NavBar from './components/NavBar/NavBar';
 import Footer from './components/Footer/Footer';
 import HomePage from './components/HomePage/HomePage';
 import Dashboard from './components/Dashboard/Dashboard';
 import PursuancePage from './components/Pursuance/PursuancePage';
 import NotFound from './components/NotFound/NotFound';
-import { removeNotification, addContributionPoints } from './actions';
+import {removeNotification, addContributionPoints} from './actions';
 import './App.css';
 
-
 class App extends Component {
   render() {
     return (
       <Router>
         <div className="App">
-          <NavBar 
-            authenticated={ this.props.authenticated } 
-            contributionPoints={ this.props.contributionPoints }
-            username={ this.props.username } 
+          <NavBar
+            authenticated={this.props.authenticated}
+            contributionPoints={this.props.contributionPoints}
+            username={this.props.username}
```

Comment:
@Spetastian But eliminating these spaces makes sense to me, but if re-adding the spaces between import curly braces requires re-adding these spaces, that's fine with me.